src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/Output/AdditionalColumnDataGenerator.cs

```diff
@@ -54,10 +60,15 @@ internal class AdditionalColumnDataGenerator : IAdditionalColumnDataGenerator
             }
             else
             {
-                return new KeyValuePair<string, object>(columnName, property.Value.ToString());
+                return new KeyValuePair<string, object>(columnName, DBNull.Value);
```

Comment:
What happens if the DB column in not nullable? Can we check this from the options of the column?